tools/Tools.Common/Utilities/ModuleFilter.cs

```diff
@@ -16,10 +16,10 @@ public static bool IsAzureStackModule(String fileName)
             }
             var executingAssemblyPath = Assembly.GetExecutingAssembly().Location;
             var versionControllerDirectory = Directory.GetParent(executingAssemblyPath).FullName;
-            var whiteListFile = Path.Combine(versionControllerDirectory, "AllowList.csv");
-            if (File.Exists(whiteListFile))
+            var allowlistFile = Path.Combine(versionControllerDirectory, "AllowList.csv");
```

Comment:
nit: should use camel-casing
```suggestion
            var allowListFile = Path.Combine(versionControllerDirectory, "AllowList.csv");
```